subxt/examples/unstable_reconnecting_rpc_client.rs

```diff
@@ -0,0 +1,161 @@
+//! Example to utilize the `reconnecting rpc client` in subxt
+//! which hidden behind behind `--feature unstable-reconnecting-rpc-client`
+//!
+//! To utilize full logs from the RPC client use:
+//! `RUST_LOG="jsonrpsee=trace,reconnecting_jsonrpsee_ws_client=trace"`
+
+#![allow(missing_docs)]
+
+use std::sync::Arc;
+use std::time::Duration;
+
+use futures::StreamExt;
+use subxt::backend::rpc::reconnecting_rpc_client::{Client, ExponentialBackoff};
+use subxt::backend::rpc::RpcClient;
+use subxt::backend::unstable::{UnstableBackend, UnstableBackendDriver};
+use subxt::error::Error;
+use subxt::tx::TxStatus;
+use subxt::{OnlineClient, PolkadotConfig};
+use subxt_signer::sr25519::dev;
+
+// Generate an interface that we can use from the node's metadata.
+#[subxt::subxt(runtime_metadata_path = "../artifacts/polkadot_metadata_small.scale")]
+pub mod polkadot {}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Nit: I wonder whether it's worth making one of these exmaples shorter, because most of the code is identical in both cases anyways.

Ideally we'd reference both of these somewhere in the book, and then we can show the "long" one first and the "short" one which just shgows the alternate backend being instantiated, second